CHANGELOG.md

```diff
@@ -108,53 +118,58 @@ argparsing with parametrized classes.
 
 ### Deprecated
 
-
 ## 0.1.9 (2021-10-20)
 
 ### Added
+
 - ([#133](https://github.com/microsoft/hi-ml/pull/133)) PyTorch Lightning logger for AzureML. Helper functions for consistent logging
 - ([#136](https://github.com/microsoft/hi-ml/pull/136)) Documentation for using low priority nodes
 
 ### Changed
+
 - ([#133](https://github.com/microsoft/hi-ml/pull/133)) Made _**large breaking changes**_ to module names,
 from `health.azure` to `health_azure`.
-- ([#144])(https://github.com/microsoft/hi-ml/pull/141) Update changelog for release and increase scope of test_register_environment to ensure that by default environments are registered with a version number
+- ([#144])(<https://github.com/microsoft/hi-ml/pull/141>) Update changelog for release and increase scope of test_register_environment to ensure that by default environments are registered with a version number
```

Comment:
Not sure why you're adding `<>`, and there's a misplaced `)` here. I believe this should read
```markdown
- ([#144](https://github.com/microsoft/hi-ml/pull/141)) ...
```